benchmark/Package.swift

```diff
@@ -129,7 +129,9 @@ targets.append(
     sources: ["main.swift"],
     swiftSettings: [.unsafeFlags(["-cxx-interoperability-mode=default",
                                   "-I",
-                                  "utils/CxxTests"])]))
+                                  "utils/CxxTests",
+                                  // FIXME rdar://136138941
```

Comment:
Could you please add a brief one-line description of the issue here in the comment?
That would help folks who don't have access to open rdar links to understand the overall issue.

Reply:
Ok sure!